lib/hooks/create/project.js

```diff
@@ -10,12 +10,15 @@ module.exports = settings => {
   const messager = Messager(settings);
 
   const needsEndorsement = (model) => {
+    const isEndorsed = get(model, 'data.meta.authority', 'no').toLowerCase() === 'yes';
+    const isTransfer = get(model, 'data.action') === 'transfer';
+    const isAmendment = get(model, 'data.modelData.status', 'inactive') === 'active';
```

Comment:
Where we have the default values in the `get` here can we remove them because they add precisely no value, but cause confusion?

(Yes, I know it was like that before)